addons/survey/static/src/scss/survey_questions_views.scss

```diff
@@ -0,0 +1,4 @@
+.o_preview_questions {
```

Comment:
When playing around on the form view on F12 and disabling those rules, I don't see much change.
Is this to add some padding simply?

Can't you use padding classes instead? (e.g: "p-1" / "p-2" / ...)?